docs/tutorials/MenuGuide.md

```diff
@@ -0,0 +1,237 @@
+Creating a simple Menu
+======================
+
+### From open.mp Wiki
+
+
+
+Contents
+--------
+
+*   1 Menus in Pawn
+*   2 First menu steps
+*   3 Add some menu items
+*   4 Creating the effects for the items
+*   5 Last steps
+*   6 Last words
+*   7 A full test script for menus
+
+
+### 1 Menus in Pawn
+-------------
+
+Menus look very complicated and difficult to script for the most players, although it isn't. Here I will show you how to create a simple menu. At the end we will have created a teleport menu.
+
+
+### 2 First menu steps
+----------------
+
+First we have to create a menu. The prefix before is Menu: this makes the variable the correct type \[there are different types for different uses such as Float: bool: Text3D: etc\] Insert this line after the #includes, #defines, forwards etc:
+
+new Menu:teleportmenu;
+
+Okay, we created the variable to store the menu. Now we have to create the menu and assign the variable we created to the menu. Type this into OnGameModeInit():
+
+teleportmenu = [CreateMenu](../scripting/functions/CreateMenu)("Teleportmenu", 2, 200.0, 100.0, 150.0, 150.0);
+
+Now a bit of an explanation about the CreateMenu arguments.
+
+[CreateMenu](../scripting/functions/CreateMenu)(title, columns, Float:x, Float:y, Float:col1width, Float:col2width);
+**Parameters:**
+|Parameter|Specifies|
+|--- |--- |
+|title|The heading of the menu
+|columns|The number here defines how much columns are used \[2 is maximum\]|
+|Float:x|The heigth position of the menu on screen \[left to right\]|
+|Float:y| The width position of the menu on screen \[up and down\]|
+|Float:col1width|The width of the first column|
+|Float:col2width|The width of the second column|
+
+
+### 3 Add some menu items
+-------------------
+
+Ok, now we've got the Menu, but we need some items, under which you can choose in the menu. You add them underneath the CreateMenu that we made earlier.
+
+[AddMenuItem](../scripting/functions/AddMenuItem)(teleportmenu, 0, "LS");
```

Comment:
This should be a code block and the URLs can be removed because they won't work in a code block anyway.